### PR TITLE
fix: Issues reported in Final Package Testing for v3.3.0.

### PR DESCRIPTION
**Pull request by RajkiranBagal**
### Description
 - Trello task - https://trello.com/c/PGLpeYu2/641-package-testing

### Types of changes
Bug fix (non-breaking change which fixes an issue)

### How has this been tested?
 - Test Cases are in the Trello Card.

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] I've included any necessary tests 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by rohitp-bsf — APPROVED** [review]

File: sass/site/builder/_header.scss [hunk 1]
```diff
@@ -657,7 +657,7 @@ body.admin-bar {
 		padding: 10px 0;
 		height: auto;
 	}
-	>*:first-child {
+	>*:first-child:not(.ast-builder-menu) {
```
Comment (Nikschavan) [inline sass/site/builder/_header.scss:660]: Can you explain what was the issue that we fixed in this?

Comment (RajkiranBagal) [reply sass/site/builder/_header.scss:660]: We had added top padding for all first child elements in the Dropdown. The top padding is not needed for Menu elements so I added the `:not`. As the Padding for Menu element does not look good.

Comment (Nikschavan) [reply sass/site/builder/_header.scss:660]: Can you give a screenshot of what exactly is the problem that we have fixed?

Comment (RajkiranBagal) [reply sass/site/builder/_header.scss:660]: Here is the Screenshot - https://a.cl.ly/qGuX9251

**Review by rohitp-bsf — APPROVED** [review]

**Review by rohitp-bsf — DISMISSED** [review]

**Review by Nikschavan — APPROVED** [review]

**Review by rohitp-bsf — APPROVED** [review]

File: assets/css/unminified/menu-animation.css [hunk 2]
```diff
@@ -1,90 +1,90 @@
...
-.ast-desktop .astra-menu-animation-fade > .menu-item .menu-item:hover > .sub-menu {
+.ast-desktop #ast-desktop-header > div:not(.ast-desktop-popup-content):not(.ast-desktop-header-content) .astra-menu-animation-fade > .menu-item.focus > .astra-full-megamenu-wrapper,
+.ast-desktop #ast-desktop-header > div:not(.ast-desktop-popup-content):not(.ast-desktop-header-content) .astra-menu-animation-fade > .menu-item:hover > .astra-full-megamenu-wrapper,
+.ast-desktop #ast-desktop-header > div:not(.ast-desktop-popup-content):not(.ast-desktop-header-content) .astra-menu-animation-fade > .menu-item.focus > .sub-menu,
+.ast-desktop #ast-desktop-header > div:not(.ast-desktop-popup-content):not(.ast-desktop-header-content) .astra-menu-animation-fade > .menu-item:hover > .sub-menu,
+.ast-desktop #ast-desktop-header > div:not(.ast-desktop-popup-content):not(.ast-desktop-header-content) .astra-menu-animation-fade > .menu-item .menu-item.focus > .sub-menu,
+.ast-desktop #ast-desktop-header > div:not(.ast-desktop-popup-content):not(.ast-desktop-header-content) .astra-menu-animation-fade > .menu-item .menu-item:hover > .sub-menu {
```
Comment (Nikschavan) [inline assets/css/unminified/menu-animation.css:80]: Can this selector be improved? this is a very large selector with two `:not()` selectors in it

Comment (RajkiranBagal) [reply assets/css/unminified/menu-animation.css:80]: Here the `:not()` selector are needed, If we eliminate those then we will need to write more extra CSS & handle each & every child CSS in it.

**Pull request merged**